### DISCUSSION TITLE
QuestPDF.Previewer

### DISCUSSION THREAD

**Discussion by lukas9812** [QuestPDF/QuestPDF#1041] [Q&A] [answered]
Hello all, 
please can somebody advice me how to use Previwer when my target framework is .NET8.0 ?
Thanks 😀

**Comment by DawidRobertNowak** [comment, marked as answer]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

The Previewer has been replaced by the Companion app, which is the current tool for previewing and debugging documents (and supports .NET 8). You can attach it to your document with `.ShowInCompanion()` during development. See the [Companion app](https://www.questpdf.com/companion/usage.html) documentation to get started.

Should you encounter any further problems, please don't hesitate to open a new issue.
Thank you!
